packages/web/src/scss/components/UNSTABLE_Avatar/index.html

```diff
@@ -0,0 +1,120 @@
+{{#> web/layout/plain }} {{setVar "sizes" "xsmall" "small" "medium" "large" "xlarge" }}
+
+<section class="docs-Section">
+  <h2 class="docs-Heading">Icon</h2>
+
+  <div class="docs-Stack docs-Stack--start">
+    <div class="d-flex" style="gap: var(--spirit-space-400)">
+
+      {{#each @root.sizes as |size|}}
+        {{setVar "iconSize" "24"}}
+        {{#if (eq size "xsmall")}}
+        {{setVar "iconSize" "16"}}
+        {{/if}}
+        {{#if (eq size "small")}}
+        {{setVar "iconSize" "20"}}
+        {{/if}}
+        {{#if (eq size "large")}}
+        {{setVar "iconSize" "28"}}
+        {{/if}}
+        {{#if (eq size "xlarge")}}
+        {{setVar "iconSize" "32"}}
+        {{/if}}
+
+        <a href="#" class="UNSTABLE_Avatar UNSTABLE_Avatar--{{size}}" title="Jiří Bárta">
```

Comment:
Does this mean that the Avatar in React should support changing element type? E.g. shouldn't the Avatar be wrapped in Link?

Reply:
🤷 so far we usually add elementType and not wrap it. I am not sure if it won't cause any problems, because I reset some link styles in the Avatar CSS

Reply:
Ok, let it be as it is. Just wonder if the composition using the Link component will be working considering using for instance a [next/link](https://nextjs.org/docs/pages/api-reference/components/link) component.

```tsx
<Link href="">
  <Avatar>
</Link>

// or

<Avatar elementType={Link} href="" />
```

I think the composition is better than using elementType but let's give this a try.

Reply:
it is as everywhere else, you would have to use the elementType. I don't know if the next/link works there, but if it works in our other components, it should work here.

Reply:
Ok.

Reply:
We will see :-)